version.php

```diff
@@ -0,0 +1,32 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Version information for the quizaccess_seb plugin.
+ *
+ * @package    quizaccess_seb
+ * @author     Andrew Madden <andrewmadden@catalyst-au.net>
+ * @copyright  2019 Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+$plugin->version = 2019120400;
+$plugin->release = 2019052000; // Match to version.
+$plugin->requires = 2017111300; // Moodle 3.7.
```

Comment:
should it be 3.9 potentially?

Reply:
I did it off the PHP version required. I'm using some 7.1 features (class constants). Just saw I copy pasted it incorrectly. Requires should be 2019052000. 

I thought it makes sense to allow people to install it as a plugin in 3.7 and 3.8 if they want to?

Reply:
Yes if we won't implement new api changes in core to support this plugin.

Reply:
Very good point. I'll try and remember to raise the version if we need to change core to integrate the plugin.